go/vt/tlstest/tlstest.go

```diff
@@ -337,12 +337,18 @@ func RevokeCertAndRegenerateCRL(root, parent, name string) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	crlList, err := x509.ParseCRL(data)
+
+	block, _ := pem.Decode(data)
+	if block == nil || block.Type != "X509 CRL" {
+		log.Fatal("failed to parse CRL PEM")
+	}
```

Comment:
`PEM` decoding is not done implicitly with `x509.ParseRevocationList` so this needs to be done here explicitly in the tests.